changelog.yaml

```diff
@@ -395,3 +395,7 @@
     fixed:
     - Chart on carbon intensities.
   date: 2022-04-08 10:35:59
+- bump: major
```

Comment:
This is definitely a huge release, but I had some backward-incompatible changes in mind before moving to 1.0, like lowercasing variable names.

Reply:
Yeah agreed - sticking to minor